<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>parcelapdv</class>
 <widget class="QDialog" name="parcelapdv">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>474</width>
    <height>498</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>parcelapdv</string>
  </property>
  <property name="styleSheet">
   <string notr="true">text-transform: uppercase;</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true">QWidget#widget{
	background-color: rgb(0, 0, 0,200);}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <property name="maximumSize">
         <size>
          <width>500</width>
          <height>500</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QWidget#widget_2{
border-radius:10px;
background-color: rgb(59, 117, 176);
}


QLabel{
color:white;
}
QDoubleSpinBox{
	font: 900 15pt &quot;Arial Black&quot;;
	border:none;
	color:white;
	background-color: transparent;
}</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="font">
            <font>
             <pointsize>25</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">
		background-color: #2b3157;
border-radius:8px;
padding:4px;
</string>
           </property>
           <property name="text">
            <string>parcela Venda</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cb_parcela">
           <property name="font">
            <font>
             <pointsize>15</pointsize>
            </font>
           </property>
           <property name="focusPolicy">
            <enum>Qt::StrongFocus</enum>
           </property>
           <property name="styleSheet">
            <string notr="true">QComboBox:on { /* shift the text when the popup opens */
    padding-top: 3px;
    padding-left: 4px;
}

QListView{
    border:                 none;
    color:                      rgb(87, 96, 134);
    background-color:   rgb(255, 255, 255);
    font-weight:            bold;
    selection-background-color: rgb(47, 175, 178);
    show-decoration-selected: 1;
    margin-left:                -10px;
    padding-left    :           15px;
}

QListView::item:hover{

    background-color:   rgb(47, 175, 178);
    border:                 none;
}</string>
           </property>
           <item>
            <property name="text">
             <string>1X</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2X</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>3X</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>4X</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>5X</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>6X</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QTableWidget" name="tb_parcela">
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="styleSheet">
            <string notr="true">QHeaderView, QHeaderView::section {
   color:white;
	background-color: #d3d7e4;
	font: 10pt &quot;Segoe UI&quot;;
color:black;
border:none;

}

QTableWidget{
	
	font: 11pt &quot;Segoe UI&quot;;
	selection-background-color: rgb(234, 110, 60);
}
</string>
           </property>
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <property name="tabKeyNavigation">
            <bool>false</bool>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::NoSelection</enum>
           </property>
           <property name="selectionBehavior">
            <enum>QAbstractItemView::SelectRows</enum>
           </property>
           <attribute name="horizontalHeaderCascadingSectionResizes">
            <bool>false</bool>
           </attribute>
           <attribute name="horizontalHeaderDefaultSectionSize">
            <number>120</number>
           </attribute>
           <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
            <bool>false</bool>
           </attribute>
           <attribute name="horizontalHeaderStretchLastSection">
            <bool>true</bool>
           </attribute>
           <attribute name="verticalHeaderVisible">
            <bool>false</bool>
           </attribute>
           <attribute name="verticalHeaderCascadingSectionResizes">
            <bool>false</bool>
           </attribute>
           <attribute name="verticalHeaderHighlightSections">
            <bool>true</bool>
           </attribute>
           <column>
            <property name="text">
             <string>data</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>valor r$</string>
            </property>
           </column>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame">
           <property name="styleSheet">
            <string notr="true">QFrame{
		background-color: #2b3157;
border-radius:8px;
padding:4px;
}
QPushButton{
padding:15px;
	font: 11pt &quot;Arial&quot;;
}</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QPushButton" name="bt_geraparcela">
              <property name="focusPolicy">
               <enum>Qt::NoFocus</enum>
              </property>
              <property name="text">
               <string>gerar (f2)</string>
              </property>
              <property name="icon">
               <iconset resource="../../icone/icone.qrc">
                <normaloff>:/venda/engrenagem.png</normaloff>:/venda/engrenagem.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>40</width>
                <height>40</height>
               </size>
              </property>
              <property name="shortcut">
               <string>F2</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="bt_imprimir">
              <property name="focusPolicy">
               <enum>Qt::NoFocus</enum>
              </property>
              <property name="text">
               <string>imprimir (ctrl+p)</string>
              </property>
              <property name="icon">
               <iconset resource="../../icone/icone.qrc">
                <normaloff>:/venda/imprimir.png</normaloff>:/venda/imprimir.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>40</width>
                <height>40</height>
               </size>
              </property>
              <property name="shortcut">
               <string>Ctrl+P</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../icone/icone.qrc"/>
 </resources>
 <connections/>
</ui>
